nautobot/utilities/testing/api.py

```diff
@@ -536,7 +536,7 @@ def test_bulk_update_objects(self):
                         obj,
                         f"Bulk update field '{field}' missing from object {i} in response",
                     )
-                    # TODO: shouldn't we also check that obj[field] == value?
+                    # TODO(Glenn): shouldn't we also check that obj[field] == value?
```

Comment:
What clarification is needed here?

Reply:
Are you suggesting I should just go ahead and make this fix while I'm sweeping these?

Reply:
No, I'm just responding to being tagged to provide clarification. :)